README.md

```diff
@@ -22,3 +22,25 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+
+- add required gems in gemfile
+
+follow getting started guide
+https://guides.rubyonrails.org/v5.0/getting_started.html#hello-rails-bang
```

Comment:
whats this doing here? we arent creating a getting started rails app?

Reply:
i was just using it as reference actually

Reply:
right okay, its kinda loosely related.
i feel like we could do the whole readme a bit better.
if you want, have a look at our other repos like `product_info_service` or `product_listing_service` and maybe write a bit more about how to start the app and how to run the tests on the app